sstable/rowblk/rowblk_64bit_test.go

```diff
@@ -0,0 +1,174 @@
+// Copyright 2025 The LevelDB-Go and Pebble Authors. All rights reserved. Use
+// of this source code is governed by a BSD-style license that can be found in
+// the LICENSE file.
+
+//go:build amd64 || arm64 || arm64be || ppc64 || ppc64le || mips64 || mips64le || s390x || sparc64 || riscv64 || loong64
+
+package rowblk
+
+import (
+	"bytes"
+	"errors"
```

Comment:
linter says to use github.com/cockroachdb/errors

Reply:
Done